gluster-exporter/main.go

```diff
@@ -95,33 +97,37 @@ func main() {
 
 	for _, m := range glusterMetrics {
 		if collectorConf, ok := conf.Collectors[m.name]; ok {
-			if collectorConf.Disabled == false {
+			if !collectorConf.Disabled {
 				go func(m glusterMetric) {
 					for {
 						m.fn()
 						interval := defaultInterval
 						if collectorConf.SyncInterval > 0 {
 							interval = time.Duration(collectorConf.SyncInterval)
 						}
-						time.Sleep(time.Duration(time.Second * time.Duration(interval)))
+						time.Sleep(time.Second * interval)
 					}
 				}(m)
 			}
 		}
 	}
 
 	if len(glusterMetrics) == 0 {
-		fmt.Fprintf(os.Stderr, "No Metrics registered, Exiting..\n")
+		_, _ = fmt.Fprintf(os.Stderr, "No Metrics registered, Exiting..\n")
 		os.Exit(1)
 	}
 
-	metricsPath, _ := conf.SystemConfig["metrics-path"]
-	port, _ := conf.SystemConfig["port"]
-	nport, _ := strconv.Atoi(port)
+	metricsPath := conf.SystemConfig["metrics-path"]
+	port := conf.SystemConfig["port"]
+	nport, err := strconv.Atoi(port)
```

Comment:
With toml based conf Atoi wont be required.

Reply:
ack